rpcs3/Emu/Audio/XAudio2/XAudio2Thread.cpp

```diff
@@ -11,11 +11,14 @@ XAudio2Thread::~XAudio2Thread()
 	if (m_source_voice) Quit();
 }
 
+XAudio2Thread::XAudio2Thread() : m_xaudio2_instance(nullptr), m_master_voice(nullptr), m_source_voice(nullptr)
+{
+}
+
 void XAudio2Thread::Init()
 {
 	HRESULT hr = S_OK;
 
-#if (FORCED_WINVER < 0x0602)
 	hr = CoInitializeEx(nullptr, COINIT_MULTITHREADED);
```

Comment:
Its nice to put the CoInitializeEx in a RAII object too - else exceptions mean CoUninitialize never happens